pkg/kubelet/cm/cgroup_manager_linux_test.go

```diff
@@ -87,14 +91,48 @@ func TestLibcontainerAdapterAdaptToSystemdAsCgroupFs(t *testing.T) {
 			input:    "/Burstable/pod_123",
 			expected: "Burstable.slice/Burstable-pod_123.slice/",
 		},
+		{
+			input:    "Burstable/pod_123",
+			expected: "Burstable.slice/Burstable-pod_123.slice/",
+		},
 		{
 			input:    "/BestEffort/pod_6c1a4e95-6bb6-11e6-bc26-28d2444e470d",
 			expected: "BestEffort.slice/BestEffort-pod_6c1a4e95_6bb6_11e6_bc26_28d2444e470d.slice/",
 		},
 	}
 	for _, testCase := range testCases {
-		f := newLibcontainerAdapter(libcontainerSystemd)
-		if actual := f.adaptName(CgroupName(testCase.input), true); actual != testCase.expected {
+		cm := NewCgroupManager(&CgroupSubsystems{}, string(libcontainerSystemd))
+		if actual := cm.Name(CgroupName(testCase.input)); actual != testCase.expected {
+			t.Errorf("Unexpected result, input: %v, expected: %v, actual: %v", testCase.input, testCase.expected, actual)
+		}
+	}
+}
+
+func TestCgroupManagerGetCgroupNameForSystemd(t *testing.T) {
+	testCases := []struct {
+		input    string
+		expected string
+	}{
+		{
+			input:    "/",
+			expected: "/",
+		},
+		{
+			input:    "Burstable.slice/",
```

Comment:
is there a code path where we pass this in outside of tests?  i am not aware of any, and would worry it was sign of a bug.